paimon-common/src/main/java/org/apache/paimon/utils/LocationKeyExtractor.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.paimon.utils;
+
+import java.util.Properties;
+
+/**
+ * declaring the name of the key in the parameters of the Hive metastore table, which indicates
+ * where the Paimon table is stored.
+ */
+public class LocationKeyExtractor {
```

Comment:
Can we merge this class into `TBPropertiesLocationHelper`?

Reply:
Another pr #886  will use this  class  for  reading  hive extern table. Shall we  migrate it to `paimon-hive-common` for a smaller scope.